README.md

```
@@ -1,3 +1,5 @@
+[![MseeP.ai Security Assessment Badge](https://mseep.net/pr/matmax-worldwide-payloadcmsmcp-badge.png)](https://mseep.ai/app/matmax-worldwide-payloadcmsmcp)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)